File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/rcm/coninfodto/d/ConInfoDTO.java
package iih.ci.rcm.coninfodto.d;

import xap.mw.core.data.BaseDTO;
import xap.mw.coreitf.d.FDate;
import xap.mw.coreitf.d.FDateTime;

/**
 * 实体 DTO数据
 * 
 */
public class ConInfoDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;

	/**
	 * 报卡名称
	 * 
	 * @return String
	 */
	public String getCard_name() {
		return ((String) getAttrVal("Card_name"));
	}

	/**
	 * 报卡名称
	 * 
	 * @param Card_name
	 */
	public void setCard_name(String Card_name) {
		setAttrVal("Card_name", Card_name);
	}

	/**
	 * 患者姓名
	 * 
	 * @return String
	 */
	public String getPat_name() {
		return ((String) getAttrVal("Pat_name"));
	}

	/**
	 * 患者姓名
	 * 
	 * @param Pat_name
	 */
	public void setPat_name(String Pat_name) {
		setAttrVal("Pat_name", Pat_name);
	}

	/**
	 * 性别
	 * 
	 * @return String
	 */
	public String getPat_xex() {
		return ((String) getAttrVal("Pat_xex"));
	}

	/**
	 * 性别
	 * 
	 * @param Pat_xex
	 */
	public void setPat_xex(String Pat_xex) {
		setAttrVal("Pat_xex", Pat_xex);
	}

	/**
	 * 诊断
	 * 
	 * @return String
	 */
	public String getDiagnose() {
		return ((String) getAttrVal("Diagnose"));
	}

	/**
	 * 诊断
	 * 
	 * @param Diagnose
	 */
	public void setDiagnose(String Diagnose) {
		setAttrVal("Diagnose", Diagnose);
	}

	/**
	 * 报卡医生
	 * 
	 * @return String
	 */
	public String getReport_doctor() {
		return ((String) getAttrVal("Report_doctor"));
	}

	/**
	 * 报卡医生
	 * 
	 * @param Report_doctor
	 */
	public void setReport_doctor(String Report_doctor) {
		setAttrVal("Report_doctor", Report_doctor);
	}

	/**
	 * 报卡日期
	 * 
	 * @return FDate
	 */
	public FDate getReport_date() {
		return ((FDate) getAttrVal("Report_date"));
	}

	/**
	 * 报卡日期
	 * 
	 * @param Report_date
	 */
	public void setReport_date(FDate Report_date) {
		setAttrVal("Report_date", Report_date);
	}

	/**
	 * 删除原因
	 * 
	 * @return String
	 */
	public String getDelete_reason() {
		return ((String) getAttrVal("Delete_reason"));
	}

	/**
	 * 删除原因
	 * 
	 * @param Delete_reason
	 */
	public void setDelete_reason(String Delete_reason) {
		setAttrVal("Delete_reason", Delete_reason);
	}

	/**
	 * 删除人员
	 * 
	 * @return String
	 */
	public String getDelete_person() {
		return ((String) getAttrVal("Delete_person"));
	}

	/**
	 * 删除人员
	 * 
	 * @param Delete_person
	 */
	public void setDelete_person(String Delete_person) {
		setAttrVal("Delete_person", Delete_person);
	}

	/**
	 * 操作时间
	 * 
	 * @return FDateTime
	 */
	public FDateTime getOpreta_time() {
		return ((FDateTime) getAttrVal("Opreta_time"));
	}

	/**
	 * 操作时间
	 * 
	 * @param Opreta_time
	 */
	public void setOpreta_time(FDateTime Opreta_time) {
		setAttrVal("Opreta_time", Opreta_time);
	}

	/**
	 * 就诊号
	 * 
	 * @return String
	 */
	public String getId_ent() {
		return ((String) getAttrVal("Id_ent"));
	}

	/**
	 * 就诊号
	 * 
	 * @param Id_ent
	 */
	public void setId_ent(String Id_ent) {
		setAttrVal("Id_ent", Id_ent);
	}

	/**
	 * 传染病id
	 * 
	 * @return String
	 */
	public String getId_contagion() {
		return ((String) getAttrVal("Id_contagion"));
	}

	/**
	 * 传染病id
	 * 
	 * @param Id_contagion
	 */
	public void setId_contagion(String Id_contagion) {
		setAttrVal("Id_contagion", Id_contagion);
	}

	/**
	 * 预留字段1
	 * 
	 * @return String
	 */
	public String getSpare_field1() {
		return ((String) getAttrVal("Spare_field1"));
	}

	/**
	 * 预留字段1
	 * 
	 * @param Spare_field1
	 */
	public void setSpare_field1(String Spare_field1) {
		setAttrVal("Spare_field1", Spare_field1);
	}

	/**
	 * 预留字段2
	 * 
	 * @return String
	 */
	public String getSpare_field2() {
		return ((String) getAttrVal("Spare_field2"));
	}

	/**
	 * 预留字段2
	 * 
	 * @param Spare_field2
	 */
	public void setSpare_field2(String Spare_field2) {
		setAttrVal("Spare_field2", Spare_field2);
	}

	/**
	 * 预留字段3
	 * 
	 * @return String
	 */
	public String getSpare_field3() {
		return ((String) getAttrVal("Spare_field3"));
	}

	/**
	 * 预留字段3
	 * 
	 * @param Spare_field3
	 */
	public void setSpare_field3(String Spare_field3) {
		setAttrVal("Spare_field3", Spare_field3);
	}

	/**
	 * 预留字段4
	 * 
	 * @return String
	 */
	public String getSpare_field4() {
		return ((String) getAttrVal("Spare_field4"));
	}

	/**
	 * 预留字段4
	 * 
	 * @param Spare_field4
	 */
	public void setSpare_field4(String Spare_field4) {
		setAttrVal("Spare_field4", Spare_field4);
	}

	/**
	 * 预留字段5
	 * 
	 * @return String
	 */
	public String getSpare_field5() {
		return ((String) getAttrVal("Spare_field5"));
	}

	/**
	 * 预留字段5
	 * 
	 * @param Spare_field5
	 */
	public void setSpare_field5(String Spare_field5) {
		setAttrVal("Spare_field5", Spare_field5);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.rcm/coninfodto/d/ConInfoDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.rcm.coninfodto.d
{
    /// <summary>
    /// ConInfoDTO 的摘要说明。
    /// </summary>
    public class ConInfoDTO : BaseDTO {

        public ConInfoDTO() {
 
        }

        /// <summary>
        /// 报卡名称
        /// </summary>
		public string Card_name {
            get { return getAttrVal<string>("Card_name",null); }
            set { setAttrVal<string>("Card_name", value); }
        }

        /// <summary>
        /// 患者姓名
        /// </summary>
		public string Pat_name {
            get { return getAttrVal<string>("Pat_name",null); }
            set { setAttrVal<string>("Pat_name", value); }
        }

        /// <summary>
        /// 性别
        /// </summary>
		public string Pat_xex {
            get { return getAttrVal<string>("Pat_xex",null); }
            set { setAttrVal<string>("Pat_xex", value); }
        }

        /// <summary>
        /// 诊断
        /// </summary>
		public string Diagnose {
            get { return getAttrVal<string>("Diagnose",null); }
            set { setAttrVal<string>("Diagnose", value); }
        }

        /// <summary>
        /// 报卡医生
        /// </summary>
		public string Report_doctor {
            get { return getAttrVal<string>("Report_doctor",null); }
            set { setAttrVal<string>("Report_doctor", value); }
        }

        /// <summary>
        /// 报卡日期
        /// </summary>
		public DateTime? Report_date {
            get { return getAttrVal<FDate>("Report_date",null); }
            set { setAttrVal<FDate>("Report_date", value); }
        }

        /// <summary>
        /// 删除原因
        /// </summary>
		public string Delete_reason {
            get { return getAttrVal<string>("Delete_reason",null); }
            set { setAttrVal<string>("Delete_reason", value); }
        }

        /// <summary>
        /// 删除人员
        /// </summary>
		public string Delete_person {
            get { return getAttrVal<string>("Delete_person",null); }
            set { setAttrVal<string>("Delete_person", value); }
        }

        /// <summary>
        /// 操作时间
        /// </summary>
		public DateTime? Opreta_time {
            get { return getAttrVal<FDateTime>("Opreta_time",null); }
            set { setAttrVal<FDateTime>("Opreta_time", value); }
        }

        /// <summary>
        /// 就诊号
        /// </summary>
		public string Id_ent {
            get { return getAttrVal<string>("Id_ent",null); }
            set { setAttrVal<string>("Id_ent", value); }
        }

        /// <summary>
        /// 传染病id
        /// </summary>
		public string Id_contagion {
            get { return getAttrVal<string>("Id_contagion",null); }
            set { setAttrVal<string>("Id_contagion", value); }
        }

        /// <summary>
        /// 预留字段1
        /// </summary>
		public string Spare_field1 {
            get { return getAttrVal<string>("Spare_field1",null); }
            set { setAttrVal<string>("Spare_field1", value); }
        }

        /// <summary>
        /// 预留字段2
        /// </summary>
		public string Spare_field2 {
            get { return getAttrVal<string>("Spare_field2",null); }
            set { setAttrVal<string>("Spare_field2", value); }
        }

        /// <summary>
        /// 预留字段3
        /// </summary>
		public string Spare_field3 {
            get { return getAttrVal<string>("Spare_field3",null); }
            set { setAttrVal<string>("Spare_field3", value); }
        }

        /// <summary>
        /// 预留字段4
        /// </summary>
		public string Spare_field4 {
            get { return getAttrVal<string>("Spare_field4",null); }
            set { setAttrVal<string>("Spare_field4", value); }
        }

        /// <summary>
        /// 预留字段5
        /// </summary>
		public string Spare_field5 {
            get { return getAttrVal<string>("Spare_field5",null); }
            set { setAttrVal<string>("Spare_field5", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Card_name", "Pat_name", "Pat_xex", "Diagnose", "Report_doctor", "Report_date", "Delete_reason", "Delete_person", "Opreta_time", "Id_ent", "Id_contagion", "Spare_field1", "Spare_field2", "Spare_field3", "Spare_field4", "Spare_field5"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.rcm.coninfodto.d.ConInfoDTO";
        }
    }
}
